README.md

```diff
@@ -146,7 +182,35 @@ class SomeController {
 
 ### Directive
 
-Static class members of directive controller are used as config directive config.
+You can configure your directive with annotation :
+
+```typescript
+@directive('ngModuleName', {
+    selector:     'atSomeDirective',                //required
+    controllerAs: 'someDirectiveCtrl',              //optional
+    templateUrl:  '/partials/some-directive.html'
+})
+class SomeDirectiveController {
+
+    public static link: angular.IDirectiveLinkFn = (scope, element, attrs, ctrl: SomeDirectiveController) => {
```

Comment:
I think isn't a best practice to declare a function into an annotation. I never see that in Java.

But I think, effectivily, the description isn't clear...